### PR TITLE
Work with blaze4d

### PR DESCRIPTION
**Pull request by ramidzkh**

### CODE REVIEW

File: src/main/java/me/hydos/rosella/render/renderer/Renderer.java [hunk 1]
```diff
@@ -228,7 +229,7 @@ public void freeSwapChain() {
         }
 
         vkDestroyRenderPass(rosella.common.device.getRawDevice(), mainRenderPass.getRawRenderPass(), null);
-        swapchain.getSwapChainImageViews().forEach(imageView ->
+        swapchain.getSwapChainImageViews().forEach((LongConsumer) imageView ->
```
Comment (hYdos) [inline src/main/java/me/hydos/rosella/render/renderer/Renderer.java]: why

**Review by hYdos — CHANGES_REQUESTED** [review]

File: src/main/java/me/hydos/rosella/render/renderer/Renderer.java (hunk 1 above)
Comment (burgerindividual) [reply src/main/java/me/hydos/rosella/render/renderer/Renderer.java]: :conc:

Comment (ramidzkh) [reply src/main/java/me/hydos/rosella/render/renderer/Renderer.java]: Ambiguous with forEach(Consumer<Long>) and forEach(LongConsumer)

**Review by hYdos — APPROVED** [review]
GLHF with the merge conflicts

File: src/main/java/me/hydos/rosella/util/HashUtil.java [hunk 2]
```diff
@@ -6,7 +6,14 @@
 
 public class HashUtil {
 
+    private static final boolean USE_XXHASH_WITH_SEED = Boolean.parseBoolean(System.getProperty("rosella:xxhash_with_seed"));
+    private static final int SEED = 0xC7AB;
```
Comment (OroArmor) [inline src/main/java/me/hydos/rosella/util/HashUtil.java]: this should be a property as well

**Review by OroArmor — CHANGES_REQUESTED** [review]

File: src/main/java/me/hydos/rosella/util/HashUtil.java (hunk 2 above)
Comment (ramidzkh) [reply src/main/java/me/hydos/rosella/util/HashUtil.java]: Why so? This is a workaround for a bug anyway

**Pull request merged**